.tasks.py

```diff
@@ -0,0 +1,52 @@
+"""Run tasks that support easy testing across multiple Python versions."""
```

Comment:
I might suggest using `scripts/tasks.py` as the location for this file, instead of making it hidden, although I think the pros and cons of each choice are up for debate.

Reply:
I think that this is a good idea and thus I have moved the file to that location. I will now update the documentation as well (assuming that I can resolve problems with AppVeyor builds!!).